parts/inc/mess

```diff
@@ -313,7 +313,7 @@ CODE:
 
 =tests plan => 102
 
-BEGIN { if ("$]" < '5.006') { $^W = 0; } }
+BEGIN { if (ivers($]) < ivers('5.006')) { $^W = 0; } }
```

Comment:
nice, maybe iV macro instead of ivers? or is it too short?

Reply:
It is a perl sub, and I think iV is too short and confusable with an IV

Reply:
good point, was just trying to find something shorter to use